docs/community/code_of_conduct.md

```diff
@@ -3,38 +3,36 @@ id: code_of_conduct
 title: Code Of Conduct
 sidebar_label: Code Of Conduct
 ---
-# Geek Beacon  Code Of Conduct
-
-
+# Geek Beacon Code Of Conduct
 
 
 # Community
 
-Geek Beacon is a community of nerds, geeks, and other enthusiasts of all types of background and interests.  We strive to be all inclusive and bigotry, hate speech and other reprehensible behavior will not be tolerated.  These are the ground rules that we try abide by in all Geek Beacon activities irrelevant of the medium used to communicate - digital, in person or otherwise.
+Geek Beacon is a community of nerds, geeks, and other enthusiasts of all types, backgrounds and interests.  We strive to be all inclusive and bigotry, hate speech and other reprehensible behavior will not be tolerated.  These are the ground rules that we try to abide by in all Geek Beacon activities irrelevant of the medium used to communicate - digital, in person or otherwise.
```

Comment:
missing a comma after background and I believe all-inclusive should have a hyphen.

Geek Beacon is a community of nerds, geeks, and other enthusiasts of all types, backgrounds, and interests. We strive to be all-inclusive and bigotry, hate speech and other reprehensible behavior will not be tolerated.

Reply:
i don't think you want a pause after all types.    I'll change the other suggestion later in.